eng/tools/TypeSpecValidation/test/tsv-test-host.ts

```diff
@@ -1,57 +1,36 @@
-import { TsvHost } from "../src/tsv-host.js";
+import { IGitOperation, TsvHost } from "../src/tsv-host.js";
 
-export function TsvTestHostGenerator(fnReturnSub: { [key: string]: Object }) {
-  let tsvTestHost = Object.assign({}, DefaultTsvTestHost); // Needs deep copy
-  for (let key in fnReturnSub) {
-    let typedKey = key as keyof TsvHost;
-    if (fnReturnSub[typedKey]) {
-      tsvTestHost = {
-        ...tsvTestHost,
-        [typedKey]: () => {
-          return fnReturnSub[typedKey];
-        },
-      };
-    } else {
-      tsvTestHost = {
-        ...tsvTestHost,
-        [typedKey]: () => {},
-      };
-    }
```

Comment:
How can we stub/mock functions in this case then for classes
for example if we want runCmd to be a function that returns true if cmd contains main.tsp and false if cmd contains client.tsp

Reply:
saw it in the other pr